roles/tas-install/tasks/prometheus_node_exporter.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: copy Helm chart to the master node
+  copy:
+    src: "{{ role_path }}/charts/node-exporter-values.yml"
+    dest: "/usr/src/charts/"
+  when:
+    - inventory_hostname == groups['kube-master'][0]
+
+- name: populate Prometheus Node Exporter Helm chart values template and push to master node
+  template:
+    src: "node-exporter-values.yml.j2"
+    dest: "/usr/src/charts/node-exporter-values.yml"
+    force: yes
+  when:
+    - inventory_hostname == groups['kube-master'][0]
+
+- name: install Prometheus Node Exporter Helm chart
+  command: helm upgrade --install --namespace {{ prometheus_node_exporter_namespace }} {{ prometheus_node_exporter_name }} -f node-exporter-values.yml stable/prometheus-node-exporter
+  args:
+    chdir: "/usr/src/charts/"
+  when:
+    - inventory_hostname == groups['kube-master'][0]
```

Comment:
this often fails on redeployment:
```
TASK [tas-install : install Prometheus Node Exporter Helm chart] **************************************************************************************************************************************************
skipping: [node1]
fatal: [master1]: FAILED! => {"changed": true, "cmd": ["helm", "upgrade", "--install", "--namespace", "kube-system", "node-exporter", "-f", "node-exporter-values.yml", "stable/prometheus-node-exporter"], "delta": "0:00:03.812645", "end": "2020-02-28 09:42:39.749309", "msg": "non-zero return code", "rc": 1, "start": "2020-02-28 09:42:35.936664", "stderr": "Error: UPGRADE FAILED: Could not get apiVersions from Kubernetes: unable to retrieve the complete list of server APIs: custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request", "stderr_lines": ["Error: UPGRADE FAILED: Could not get apiVersions from Kubernetes: unable to retrieve the complete list of server APIs: custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request"], "stdout": "UPGRADE FAILED\nError: Could not get apiVersions from Kubernetes: unable to retrieve the complete list of server APIs: custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request", "stdout_lines": ["UPGRADE FAILED", "Error: Could not get apiVersions from Kubernetes: unable to retrieve the complete list of server APIs: custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request"]}
```

purging all Prometheus Helm deployments before rerunning deployment seems to solve it, so maybe this is a way to fix it